### DISCUSSION TITLE
When you move a page, the modification isn't applied globally

### DISCUSSION THREAD

**Discussion by tigerblue77** [requarks/wiki#3309] [Error / Bug Report]
**Describe the bug**
If I have a page "toto" whose path is "/toto" (for example) and I change the path to "/tata" then I have to edit each menu and link previously pointing on it...

**To Reproduce**
1. Create a page
2. Create a menu pointing to this page
3. Test your menu
4. Edit the path of this page
5. Test your menu again (broken)

**Expected behavior**
I expected all the paths to my page to be automatically updated

**Host Info (please complete the following information):**
- OS: Windows 10 Professionnel Insider Preview (20H2 Build 21292.1000)
- Wiki.js version: 2.5.170
- Database engine: Postgres 13

**Comment by btittelbach** [comment]
Coming from other wiki's, I had the same expectation. Does not seem to work though.
My guess is, that the different supported formats make this more complex.

I would definitely be very much in favour of implementing this feature and argue that it has a high priority. (Can features be sponsored?)
We found that, especially in the beginning, the structure and paths of our wiki changes often, which leaves maintaining
the always broken links a challenge.

Maybe it's already planned for 3.0? Though I could not find anything on that.

**Reply by ghost** [reply]
It looks like it's planned https://requarks.canny.io/wiki/p/moving-a-page-should-update-links but a estimated date for an implementation would be nice or if someone needs help implementing it.

**Reply by tigerblue77** [reply]
I will stop using this wiki for the moment, changes are too slow and too many bugs/incoherences.
